Package.swift

```diff
@@ -16,7 +16,9 @@ let package = Package(
   products: [
     .executable(name: "protoc-gen-swift", targets: ["protoc-gen-swift"]),
     .library(name: "SwiftProtobuf", targets: ["SwiftProtobuf"]),
+    .library(name: "SwiftProtobufDynamic", type: .dynamic, targets: ["SwiftProtobuf"]),
     .library(name: "SwiftProtobufPluginLibrary", targets: ["SwiftProtobufPluginLibrary"]),
+    .library(name: "SwiftProtobufPluginLibraryDynamic", type: .dynamic, targets: ["SwiftProtobufPluginLibrary"]),
```

Comment:
The `SwiftProtobufPluginLibrary` does not have a stable API or ABI.  We should certainly not provide a dynamic version of that at this time.